src/test/java/CheckboxElementsTest.java

```diff
@@ -0,0 +1,78 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class CheckboxElementsTest {
+    WebDriver driver;
+    Utils utils;
+    List<WebElement> checkboxElements;
+
+    @BeforeMethod
+    public void initTest(){
+        utils = new Utils();
+        utils.setPropertyDriver();
+
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        //open site
+        utils.openSite(driver, Constants.Checkboxes.URL_ELEMENTS);
+    }
+
+    @Test
+    public void uncheckedFirstTest(){
+        //init elements
+        initCheckboxElements();
+        checkboxTest(checkboxElements.get(0), false, false);
+    }
+
+    @Test
+    public void checkedFirstTest(){
+        //init elements
+        initCheckboxElements();
+        checkboxTest(checkboxElements.get(0), true, true);
+    }
+
+    @Test
+    public void checkedSecondTest(){
+        //init elements
+        initCheckboxElements();
+        checkboxTest(checkboxElements.get(1), true, false);
+    }
+
+    @Test
+    public void uncheckedSecondTest(){
+        //init elements
+        initCheckboxElements();
+        checkboxTest(checkboxElements.get(1), false, true);
+    }
+
+    @AfterMethod
+    public void exitTest(){
+        utils.quitDriver(driver);
+    }
+
+    public void checkboxTest(WebElement checkbox, boolean isSelected, boolean isClick){
```

Comment:
better to rename to smth like that: 'isCheckboxCheckedTest' to simply understand what this method is doing inside.

Reply:
done